doc/source/management/backup_locks.rst

```diff
@@ -19,23 +19,13 @@ If an "unsafe" statement is executed in the same connection that is holding a ``
 
  UNLOCK TABLES releases the lock acquired by LOCK TABLES FOR BACKUP.
 
-``LOCK BINLOG FOR BACKUP``
----------------------------
-
-``LOCK BINLOG FOR BACKUP`` uses another new MDL lock type to block all operations that might change either binary log position or ``Exec_Master_Log_Pos`` or ``Exec_Gtid_Set`` (i.e. master binary log coordinates corresponding to the current SQL thread state on a replication slave) as reported by ``SHOW MASTER``/``SLAVE STATUS``. More specifically, a commit will only be blocked if the binary log is enabled (both globally, and for connection with sql_log_bin), or if commit is performed by a slave thread and would advance ``Exec_Master_Log_Pos`` or ``Executed_Gtid_Set``. Connections that are currently blocked on the global binlog lock can be identified by the ``Waiting for binlog lock`` status in ``PROCESSLIST``.
-
-.. _backup-safe_binlog_information:
-
-``LOCK TABLES FOR BACKUP`` flushes the current binary log coordinates to |InnoDB|. Thus, under active ``LOCK TABLES FOR BACKUP``, the binary log coordinates in |InnoDB| are consistent with its redo log and any non-transactional updates (as the latter are blocked by ``LOCK TABLES FOR BACKUP``). It is planned that this change will enable |Percona XtraBackup| to avoid issuing the more invasive ``LOCK BINLOG FOR BACKUP`` command under some circumstances.
-
 ``UNLOCK BINLOG``
 ------------------
 
-``UNLOCK BINLOG`` releases the ``LOCK BINLOG FOR BACKUP`` lock, if acquired by the current connection. The intended use case for |Percona XtraBackup| is: :: 
+The intended use case for |Percona XtraBackup| is: :: 
```

Comment:
The use case does not work anymore if LBFB is removed. Please consult with @gl-sergei for how it should look like